### ISSUE TITLE
Urgent review - 360 view on Athletes for advisors

### ISSUE THREAD

**Issue by ChrisImagineer** [gordon-cs/gordon-360-ui#2139]

**Comment by EjPlatzer** [comment]
Hi @ChrisImagineer can you send this in as a ticket to CTS please?

**Comment by Luke-W-Hart** [comment]
@EjPlatzer I'm assuming this can be closed because it was sent in as a ticket?

**Comment by EjPlatzer** [comment]
> @EjPlatzer I'm assuming this can be closed because it was sent in as a ticket?

@Luke-W-Hart No, this has not been resolved (no ticket was ever created), so please do not close it. Thanks!
